spec/support/support.rb

```diff
@@ -0,0 +1,16 @@
+# Namespace that handles git operations for PackageProvider
+module PackageProvider
+  module Spec
+    # Helper module with methods to support unit and integratin tests
+    class Helpers
```

Comment:
use module, it is not necessary to put it in PackageProvider::Spec. See https://www.relishapp.com/rspec/rspec-core/v/3-3/docs/helper-methods/define-helper-methods-in-a-module and https://www.relishapp.com/rspec/rspec-core/v/3-3/docs/helper-methods/arbitrary-helper-methods